packages/infrastructure/kube_nats_jetstream/main.tf

```diff
@@ -0,0 +1,256 @@
+terraform {
+  required_providers {
+    kubernetes = {
+      source  = "hashicorp/kubernetes"
+      version = "2.27.0"
+    }
+    kubectl = {
+      source  = "alekc/kubectl"
+      version = "2.0.4"
+    }
+    helm = {
+      source  = "hashicorp/helm"
+      version = "2.12.1"
+    }
+  }
+}
+
+locals {
+
+  name             = "jetstream-testing"
+  namespace        = module.namespace.namespace
+  username         = "client"
+  cluster_username = "user"
+  js_username      = "js-user"
+  pw               = "password"
+
+}
+
+module "namespace" {
+  source    = "github.com/Panfactum/stack.git//packages/infrastructure/kube_namespace?ref=dfa23313b6748c957f95af6eb1a322e0fe170f12" # pf-update
+  namespace = local.name
+
+  # pf-generate: pass_vars
+  pf_stack_version = var.pf_stack_version
+  pf_stack_commit  = var.pf_stack_commit
+  environment      = var.environment
+  region           = var.region
+  pf_root_module   = var.pf_root_module
+  is_local         = var.is_local
+  extra_tags       = var.extra_tags
+  # end-generate
+}
+
+resource "helm_release" "nats" {
+  namespace       = local.namespace
+  name            = "nats"
+  repository      = "oci://registry-1.docker.io/bitnamicharts"
+  chart           = "nats"
+  version         = "8.4.5"
+  recreate_pods   = false
+  cleanup_on_fail = true
+  wait            = true
+  wait_for_jobs   = true
+
+  # The NATS configuration below is being overwritten by the nats-server.conf in this folder. This was termporary test getting a connection to NATS up
+  values = [
+    yamlencode({
+      global = {
+        defaultStorageClass = "ebs-standard"
+      }
+      auth = {
+        enabled  = true
+        user     = local.username
+        password = local.pw
+      }
+      cluster = {
+        auth = {
+          enabled  = true
+          user     = local.cluster_username
+          password = local.pw
+        }
+      }
+      jetstream = {
+        enabled   = true
+        maxMemory = "1G"
+      }
+      persistence = {
+        enabled = true
+        size    = "1Gi"
+      }
+      replicaCount = 3
+      tolerations = [
+        {
+          key      = "arm64"
+          operator = "Equal"
+          value    = "true"
+          effect   = "NoSchedule"
+        },
+        {
+          key      = "spot"
+          operator = "Equal"
+          value    = "true"
+          effect   = "NoSchedule"
+        },
+        {
+          key      = "burstable"
+          operator = "Equal"
+          value    = "true"
+          effect   = "NoSchedule"
+        }
+      ]
+      configuration = file("nats-server.conf")
```

Comment:
You this file contradicts the values you are passing to the helm chart (e.g., auth). This makes it difficult to discern what your intention is.